README.md

```diff
@@ -1,4 +1,8 @@
 # EarlGrey
+[![Apache License](https://img.shields.io/badge/license-Apache%202-lightgrey.svg?style=flat)](https://github.com/google/EarlGrey/blob/master/LICENSE)
+[![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
```

Comment:
Yup. We should add a Carthage badge only when we provide a "Carthage Installation Steps" guide in the docs.